pkgs/by-name/wh/wheelwizard/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  makeWrapper,
+  makeDesktopItem,
+  desktop-file-utils,
+}:
+buildDotnetModule rec {
+  pname = "wheelwizard";
+  version = "2.3.3";
+
+  src = fetchFromGitHub {
+    owner = "TeamWheelWizard";
+    repo = "WheelWizard";
+    tag = "${version}";
+    hash = "sha256-DuEI6bmvNP6wRuZX9Do0FGDsu80ldy0SCefBk6gqT9s=";
+  };
+  postPatch = ''
+    rm .config/dotnet-tools.json
+  '';
+
+  projectFile = "WheelWizard.sln";
+  buildType = "Release";
+  dotnet-sdk = dotnetCorePackages.sdk_8_0-bin;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0-bin;
+  dotnet-tools = [ ];
+  nugetDeps = ./deps.json;
+
+  nativeBuildInputs = [
+    makeWrapper
+    desktop-file-utils
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib/${pname}
+
+    cp -r WheelWizard/bin/Release/net8.0/* $out/lib/${pname}/
+
+    mkdir -p $out/bin
```

Comment:
minor nit, but I personally prefer to init the out dirs together before further processing.